app/scripts/controllers/edit/route.js

```diff
@@ -126,6 +127,13 @@ angular.module('openshiftConsole')
             _.set(updated, 'spec.to.weight', weight);
           }
 
```

Comment:
I'm tempted to want to see this bit function broken into smaller pieces instead of making it longer, but given you mentioned this is going to be backported, I can be ok with it 😄